include/xten/Conversion/ATenToXTen.td

```diff
@@ -29,8 +29,11 @@ def CheckFalse : Constraint<
 
 // Conversions
 
-def : Pat<(Torch_AtenConv2dOp $a,$b,$c,$d,$e,$f,$g),
-          (XTen_Conv2dOp  $a,$b,$c,$d,$e,$f,$g)>;
+// def : Pat<(Torch_AtenConv2dOp $a,$b,$c,$d,$e,$f,$g),
+//           (XTen_Conv2dOp  $a,$b,$c,$d,$e,$f,$g)>;
+
+def : Pat<(Torch_AtenConvolutionOp $input,$weight,$bias,$stride,$padding,$dilation,$transposed,$output_padding,$groups),
```

Comment:
This still needs a check that `$transposed` is `false` and `$output_padding` is an empty list.